kalihealthcheck.sh

```diff
@@ -1,54 +1,109 @@
+#!/usr/bin/env bash
+
+export logfile='/tmp/kalicheck.txt'
+export update=('apt update && apt upgrade' 'apt-get update && apt-get dist-upgrade')
+export repos=('/etc/apt/sources.list')
+export docs=('http://docs.kali.org/general-use/kali-linux-sources-list-repositories')
+export kernel=("$(uname -v | awk '{print$4}')" "curl -sL http://pkg.kali.org/linux | awk '/version:/ {getline;print$1}'")
 clear
-export logfile=/tmp/kalicheck.txt
```

Comment:
Not sure why you used `export`, so I just followed suit with a bunch of variables. 
Some of them aren't even implemented yet.